### PR TITLE
refactor: separate abaci tests

### PR DESCRIPTION
**Pull request by kmbarry1**

### CODE REVIEW

**Review by lucas-manuel — APPROVED** [review]
LGTM

**Comment by lucas-manuel** [comment]
@kmbarry1 I also think that this can be merged after one approval.

**Comment by kmbarry1** [comment]
> @kmbarry1 I also think that this can be merged after one approval.

Haha I agree, the PR process exception clause is exercised for the first time :)

**Pull request merged**
